ext/strings.go

```diff
@@ -44,6 +56,42 @@ import (
 //	'hello'.charAt(5)  // return ''
 //	'hello'.charAt(-1) // error
 //
+// # Format
+//
+// Returns a new string with substitutions being performed, printf-style.
```

Comment:
Any other types from https://github.com/google/cel-spec/blob/master/doc/langdef.md#values that should be supported? `boolean`? `type`?

Reply:
(marking as resolved now that `%s` handles lists and maps and `%t` handles booleans)